pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+markers =
+    gt_3_7: Mark a test as requiring python > 3.7.
+    gt_3_8: Mark a test as requiring python > 3.8.
+    gt_3_9: Mark a test as requiring python > 3.9.
```

Comment:
Do we need to add "gt_3.10" as well? I noticed it is mentioned in the README

Reply:
Curiously not: the logic of `-k not some_mark` is really `if some_mark not in test.marks`, so it will actually work fine.  However I'll add a mark for clarity.  The idea here was just to test the new dict syntax in python 3.9 to check lru caches really behave like dicts.